client/package.json

```diff
@@ -31,6 +31,8 @@
     "glob": "^7.1.4",
     "handsontable": "^2.0.0",
     "imask": "^4.1.5",
+    "imports-loader": "^0.8.0",
```

Comment:
Not sure what went sideways here, but this should only be in devDependencies.  Same as `webpack-dev-server` below.

Reply:
Thanks! it was most probably a merge problem. it should be better now.